syncserver/__init__.py

```diff
@@ -155,7 +159,9 @@ def includeme(config):
 
     # Include the relevant sub-packages.
     config.scan("syncserver", ignore=["syncserver.wsgi_app"])
-    config.include("syncstorage", route_prefix="/storage")
+
+    subprocess.call("%sSYNC_MASTER_SECRET=%s /usr/bin/env sh run-syncstorage-rs.sh &" % (sync_database_url, secret, ), shell=True)
+
```

Comment:
Really wish shells were more consistent about how they recognize ENV vars in CLI, but that's a personal thing.

Reply:
Please, if you have a better way to do this, let's do it. I suppose I could set the environment in the parent, then call the child, instead?

Reply:
IIRC you can pass `env={"SYNC_MASTER_SECRET": secret, "SYNC_DATABASE_URL": sync_database_url}` as an argument to `subprocess.call()`. I'd have to experiment a bit to verify (I think there are problems on Windows, but I'm not 100% sure we care).

Reply:
Setting os.environ in the parent before subprocess.call forked the child worked.